### PR TITLE
feat: Introduces docker, maven and npm registry resources

### PR DESCRIPTION
**Pull request by hofmeister**

### CODE REVIEW

File: kapeta.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  description: Provides a maven repository resource to your block
+  visibility: public
+spec:
+  icon:
+    type: url
+    value: https://storage.googleapis.com/kapeta-public-cdn/icons/maven.svg
+  ports:
+    - type: http
+  local:
+    image: dzikoysk/reposilite:3.4.6
```
Comment (sorenmat) [inline kapeta.yml:61]: perhaps we should mirrors these images??

Comment (hofmeister) [reply kapeta.yml:61]: We dont really elsewhere though ? You're thinking so they dont suddenly disappear ?

Comment (sorenmat) [reply kapeta.yml:61]: Exactly

Comment (hofmeister) [reply kapeta.yml:61]: This live on dockerhub - so should be ok. But add a ticket for it ? It's a good idea - we then just need to be consistent imo

**Pull request merged**